pkg/globalnet/controllers/ipam/constants.go

```diff
@@ -4,3 +4,17 @@ import "time"
 
 const handlerResync = time.Hour * 24
 const submarinerIpamGlobalIp = "submariner.io/globalIp"
+const submarinerGlobalNet = "SUBMARINER-GLOBALNET"
+
+const kubeProxyServiceChainPrefix = "KUBE-SVC-"
```

Comment:
explain that this comes from k8s and link to the origin (to make easier to fix if something breaks because they changed something)